cypress/e2e/tests/pages/users-and-auth/roles.spec.ts

```diff
@@ -383,6 +391,9 @@ describe('Roles Templates', { tags: ['@usersAndAuths', '@adminUser'] }, () => {
     });
 
     it('sorting changes the order of paginated global roles data', () => {
+      cy.tableRowsPerPage(10);
+      cy.reload(true);
+      usersPo.goTo();
```

Comment:
can remove once above is addressed

Reply:
https://github.com/rancher/dashboard/blob/56f7893b89e1251a42ba1bf9925f8df6308721ca/cypress/e2e/tests/pages/users-and-auth/roles.spec.ts#L261